packages/aws-cdk/lib/api/util/sdk.ts

```diff
@@ -53,6 +57,20 @@ export interface SDKOptions {
    * @default Automatically determine.
    */
   ec2creds?: boolean;
+
+  /**
+   * Change the endpoint addresses of the aws sdk services
+   *
+   * The argument supports both comma separated serviceName=URL pairs,
+   * and JSON structures
+   * E.g.
+   * cdk deploy --endpoints 'cloudformation=http://localhost:4581,s3=http://localhost:4572'
+   * cdk deploy --endpoints '{"cloudformation":"http://localhost:4581","s3":"http://localhost:4572"}'
+   *
+   * @default Online AWS cloud endpoints
```

Comment:
I would call those *standard*, not *online*.

Also, it could be nice to allow users to specify these from a file, for example using a value prefixed by an `@`@. as is done by certain other tools (`--endpoints @./local-endpoints.json`).

Reply:
Hmm..

I must admit that i think its a lot of extra work for something that can be done through `--endpoints $(cat  ./local-endpoints.json)`, as your colleague suggested.

Regarding standard, i totally agree with you.